creator-node/src/premiumContent/types.ts

```diff
@@ -0,0 +1,8 @@
+export type PremiumContentType = 'track'
```

Comment:
nitty/preference thing - imo we could just colocate this in the file where it's used